src/libm/sleefsimddp.c

```diff
@@ -2407,6 +2407,9 @@ EXPORT CONST vdouble xfma(vdouble x, vdouble y, vdouble z) {
 }
 
 EXPORT CONST vdouble xsqrt_u05(vdouble d) {
+#ifdef ACCURATE_SQRT
+  return vsqrt_vd_vd(d);
+#else
```

Comment:
Is there an helper file that does not define ACCURATE_SQRT? If not, we should remove the code in this branch, as it is not used.

Reply:
I want to keep the code since I want to claim that all the functions in the C99 standard are implemented.

Reply:
> I want to keep the code since I want to claim that all the functions in the C99 standard are implemented

OK, but we need to make sure we are testing the polynomial approximation. I think you should define a new function then.

xsqrt: uses the instruction
xsqrt_05: uses the polynomial approximation.

Both functions need to be tested with the same tests.

The xsqrt one is the one that needs to be exported as _ZGV*_sqrt when /* ACCURATE_SQRT */ is defined.

Reply:
But then how should those functions be implemented if sqrt instruction is not available?
You know, aarch32 does not have a vector sqrt instruction. Availability of vector sqrt operation is not assumed in helpervecext.h or helperpurec.h.

The style in sleef is basically to implement everything with software, but if some unique feature is available in an extension, use it. In this case, sqrt instruction is viewed as a feature of vector extension.